README.md

```diff
@@ -347,6 +347,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [Tox](https://tox.chat/) - Distributed, secure messenger with audio and video chat capabilities. ([Source Code](https://github.com/TokTok/c-toxcore)) `GPL-3.0` `C`
 - [Tuber](https://blog.trailofbits.com/2015/12/15/self-hosted-video-chat-with-tuber/) - Peer-to-peer video chat that works. ([Source Code](https://github.com/trailofbits/tubertc)) `MIT` `Javascript`
 - [Typebot](https://typebot.io) - Typebot is a conversational app builder as an alternative to Typeform or Landbot. ([Source Code](https://github.com/baptisteArno/typebot.io)) `AGPL-3.0` `Docker`
+- [VoceChat](https://voce.chat) - Private social sphere like Slack/Discord written with Rust. Host anywhere, embed anywhere. VoceChat is for you.  ([Demo](https://voce.chat)) `MIT` `Rust/Docker/Web/iOS/Android`
```

Comment:
Quite a few changes to be made
- Remove the duplicate demo link 
- Remove the second space after `for you` and before demo
- Remove `/Web/iOS/Android` from the language tag